src/main/java/com/github/pires/obd/commands/control/DtcNumberCommand.java

```diff
@@ -45,8 +45,7 @@ protected void performCalculations() {
      */
     public String getFormattedResult() {
         final String res = milOn ? "MIL is ON" : "MIL is OFF";
-        return new StringBuilder().append(res).append(codeCount).append(" codes")
-                .toString();
+        return res + codeCount + " codes";
```

Comment:
@bbarnhill you are right and I am truly sorry. I did think about requesting @valens254 to keep using `StringBuilder` for concatenation, but his PR was not just about that, like yours. So in the end, I just merged.
TBH, I'm not even sure why I cared so much about _performance_. Again, sorry!